.github/workflows/validate-upstream.yml

```diff
@@ -41,24 +41,29 @@ jobs:
           path: /tmp/data/${{ inputs.data-files-folder }}
       -
         # Copy data files from /tmp/data/${{ inputs.data-files-folder }} to
-        # data/${{ inputs.data-files-folder }}. If data-files-placeholder-folder
-        # is set, then check if a placeholder file exists for each data file in
-        # that folder. If not, then creates a placeholder file with the same
-        # name as the data file, but with a .md extension.
+        # data/${{ inputs.data-files-folder }}. If create-placeholder-stubs
+        # is set to true, then check if a placeholder file exists for each data file in
+        # that folder. If not, create a placeholder stub file for the data file.
         name: Copy data files
         if: ${{ inputs.data-files-id != '' && inputs.data-files-folder != '' }}
         uses: actions/github-script@v7
         with:
           script: |
             const fs = require('fs');
             const path = require('path');
-            const dataFilesPlaceholderFolder = `content/${{ inputs.data-files-placeholder-folder }}`;
+            const glob = require('glob');
```

Comment:
Didn't see that we "required" this package before so maybe this isn't necessary, but threw it in just in case.

Reply:
Not needed to require glob, it's already included per what's shown in their README: https://github.com/actions/github-script?tab=readme-ov-file#actionsgithub-script